### PR TITLE
Chirp Button Component Created

### PR DESCRIPTION
**Pull request by uzoh**
### What does this PR do?
- This PR Provides a Button Component 

### Description of tasks to be completed
- Create of Button Component for a custom Chirp Button

### How to manually test
- when the button is clicked, the isLoading is set to true and vice versa

### Any background context you want to provide
- It provides a beautiful animation when it is loading 

### Screenshots
Simulator Screen Recording - iPhone 14 Pro - 2023-06-06 at 15 38 39

### CODE REVIEW

File: Chirp/Components/Button.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        setup()
+    }
+    
+    private func setup() {
+        addSubview(loadingView)
+        backgroundColor = UIColor(named: "app-brown")
+        clipsToBounds = true
+        layer.cornerRadius = 6
+        
+        contentEdgeInsets = .init(top: 10, left: 10, bottom: 10, right: 10)
```
Comment (EMacco) [inline Chirp/Components/Button.swift:41]: Can we create a constant at the top of the file to hold this value?
```suggestion
        contentEdgeInsets = .init(top: buttonPadding, left: buttonPadding, bottom: buttonPadding, right: buttonPadding)
```

File: Chirp/Components/Button.swift [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        setup()
+    }
+    
+    private func setup() {
+        addSubview(loadingView)
+        backgroundColor = UIColor(named: "app-brown")
```
Comment (EMacco) [inline Chirp/Components/Button.swift:37]: Same feedback as the `app-brown-secondary` implementation

File: Chirp/ViewController.swift [hunk 3]
```diff
@@ -12,8 +12,7 @@ class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
+        
     }
-
-
```
Comment (EMacco) [inline Chirp/ViewController.swift:17]: Maybe next time we don't push changes that are not needed

**Review by EMacco — COMMENTED** [review]
Great work on this!!! 🔥🔥🔥
I've left a few comments. Please implement and let me know for final review

**Review by EMacco — APPROVED** [review]
Great work man! 🔥

**Review by stepharski — APPROVED** [review]

**Pull request merged**